packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Timeline/Reflow/DynamicData.2.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import DynamicData from '@cerner/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/Timeline/Reflow/DynamicData.jsx';
+import PropsTable from '@cerner/terra-graphs/lib/components/Timeline/Timeline?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelineConfigObject?dev-site-codeblock';
+import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Timeline/defaultDataset.js?dev-site-codeblock';
```

Comment:
Are we using the same dataset for all the graphs?

Reply:
Yeah. I also added the pandata here: 5f101bc17a20c4113abb0d38194407e62c01757f